static/app/components/profiling/flamegraph/flamegraphLayout.tsx

```diff
@@ -264,10 +264,10 @@ const FlamegraphGrid = styled('div')<{
   width: 100%;
   grid-template-rows: ${({layout}) =>
     layout === 'table bottom'
-      ? 'auto auto auto 1fr'
+      ? 'auto auto auto auto 1fr'
```

Comment:
Will this cause issues with platforms that don't have CPU charts?

Reply:
I tested it and seems like it just doesn't do anything if the element that takes the slot is missing. Iirc UI frames falls into the same bucket. I'll test again to make sure